package.json

```diff
@@ -105,6 +105,7 @@
     "fast-json-patch": "^2.0.4",
     "fuse.js": "^3.2.0",
     "gaba": "^1.6.0",
+    "gulp-terser-js": "^5.0.0",
```

Comment:
This should be a `devDependency` rather than a production dependency.

Well, since this isn't a library, the distinction isn't quite as meaningful. But we seem to have been pulling all build tools under `devDependencies` by convention.

Reply:
nice catch

Reply:
its useful for things like `yarn list --production`